File: src/main/angular/agendasProject/src/app/dto/AgendaItemDto.ts
export class AgendaItemDto{

    public id: number | null;
    public itemOrder: number | null;
    public phase: string | null;
    public content: string | null;
    public objectives: string | null;
    public duration: number | null;
    public creditable: boolean;

	constructor(){
		this.id = null;
        this.itemOrder = null;
		this.phase = null;
		this.content = null;
        this.objectives = null;
        this.duration = null;
        this.creditable = false;
	}
}

File: src/main/java/com/netdimen/agendaeditor/agenda/model/dto/AgendaItemDto.java
package com.netdimen.agendaeditor.agenda.model.dto;

import lombok.Data;
import lombok.NoArgsConstructor;
import lombok.NonNull;

@Data
@NoArgsConstructor
public class AgendaItemDto {
    private Long id;
    @NonNull private int itemOrder;
    @NonNull private String phase;
    private String content;
    private String objectives;
    @NonNull private Long duration;
    private boolean creditable;
}
